pkg/controller/node/nodecontroller.go

```diff
@@ -67,10 +67,6 @@ var (
 	ErrCloudInstance        = errors.New("cloud provider doesn't support instances.")
 	gracefulDeletionVersion = utilversion.MustParseSemantic("v1.1.0")
```

Comment:
Also remove this v1.1.0 thing? And an associated TODO here: https://github.com/k82cn/kubernetes/blob/f6161bffdc9436edc92d435fb2c5258d40ec567a/pkg/controller/node/controller_utils.go#L177